src/platform/AT32/timer_at32bsp.c

```diff
@@ -677,7 +677,7 @@ void timerInit(void)
     memset(timerConfig, 0, sizeof(timerConfig));
 
 #if defined(PARTIAL_REMAP_TIM3)
-    GPIO_PinRemapConfig(GPIO_PartialRemap_TIM3, ENABLE);
+    GPIO_PinRemautopilotConfig(GPIO_PartialRemap_TIM3, ENABLE);
```

Comment:
`apConfig` of `RemapConfig` accidentally got replaced
```suggestion
    GPIO_PinRemapConfig(GPIO_PartialRemap_TIM3, ENABLE);
```

Reply:
thanks, fixed